app/controllers/concerns/contact_attributes.rb

```diff
@@ -3,15 +3,28 @@ module ContactAttributes
 
   included do
     def contact_attributes
-      [:id, :first_name, :last_name, :_destroy, :contactable_id, :contactable_type, :street,
-      :extended, :city, :postal_code,
-      contact_emails_attributes: contact_point_attrs,
-      contact_phones_attributes: contact_point_attrs
-    ]
+      {
+        contact_attributes:
+        [
+          :id, :first_name, :last_name, :_destroy, :contactable_id,
+          :contactable_type, :street, :extended, :city, :postal_code,
+          :contact_emails_attributes, :contact_phones_attributes,
+        ]
+      }
+    end
+
+    def contact_emails_attributes
+      { contact_emails_attributes: contact_point_attrs }
+    end
+
+    def contact_phones_attributes
+      { contact_phones_attributes: contact_point_attrs }
     end
 
     def contact_point_attrs
-      [:id, :body, :label, :_destroy, :type, :contacts_id]
+      {
+        contact_point_attrs: [:id, :body, :label, :_destroy, :type, :contacts_id]
+      }
     end
```

Comment:
Exactly, but contact_point_attrs is an exception because it will be applied to contact_emails_attributes: and contact_phones_attributes. But for any other attributes bundle,  it's exactly like that.